src/Rfc6455Client.php

```diff
@@ -146,7 +146,8 @@ public function getIterator(): \Iterator
 
     public function receive(?Cancellation $cancellation = null): ?Message
     {
-        return $this->messagePipeline->continue($cancellation);
+        // TODO this method no longer exists
```

Comment:
The `Pipeline::continue()` method was removed in amphp/pipeline#10 - not sure how to handle it with this method.

Reply:
You can use getIterator and call continue on that